src/bp/ui-studio/src/web/components/Layout/Sidebar.jsx

```diff
@@ -159,21 +161,11 @@ class Sidebar extends React.Component {
       </div>
     )
 
-    const isOpen = this.props.viewMode < 1
-
     return (
-      <ReactSidebar
-        sidebarClassName={classnames(style.sidebarReact, 'bp-sidebar-react')}
-        sidebar={sidebarContent}
-        open={isOpen}
-        docked={isOpen}
-        shadow={false}
-        transitions={false}
-        styles={{ sidebar: { zIndex: 20 } }}
-        onSetOpen={this.onSetSidebarOpen}
-      >
+      <aside>
+        {sidebarContent}
```

Comment:
I know you are not the one that wrote this, but can you remove this extracted variable and simply add the markup here. I mean, to have it extracted simply decrease readability and makes a completely useless memory assignment (I'm being picky here, I know).